change/@ni-nimble-components-e878875a-13bb-474b-b54e-76adb87bc3fa.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
The ctrl behavior is a little surprising to me. I see the following:
1. Click on a row
2. ctrl+ click on a different row to add to my selection instead of change
3. ctrl+shift+click on a different row. Expected: adds that range. Actual: Just adds the single row

https://60e89457a987cf003efc0a5b-fohguihpuy.chromatic.com/?path=/story/components-table--table&args=data:LargeDataSet;selectionMode:multiple

![rangectrlclick](https://github.com/ni/nimble/assets/1588923/c9f5282b-ee7d-41a3-9c88-5ea5107751a8)

Reply:
I believe that is existing behavior. I was able to reproduce it on our published storybook. I think it'll be as simple as switching the order of two if-statements in the code to fix (i.e. look for the `shift` case before looking for the `ctrl` case). If so, I can address that in this PR as well.